packages/components/src/notice/types.ts

```diff
@@ -115,6 +115,12 @@ export type NoticeProps = {
 	 * instead of a string.
 	 */
 	__unstableHTML?: boolean;
+	/**
+	 * How long the notice should be presented to users
+	 *
+	 * @default 10000
+	 */
+	duration?: number;
```

Comment:
Why do we need this? What does `Notice` have to do with `Snackbar` and why does this also need this prop?